dotnet/Directory.Packages.props

```diff
@@ -60,8 +60,8 @@
     <PackageVersion Include="Microsoft.Graph" Version="[4.51.0, 5)" />
     <PackageVersion Include="Microsoft.Azure.Cosmos" Version="[3.32.3, )" />
     <PackageVersion Include="Microsoft.Identity.Client.Extensions.Msal" Version="[2.28.0, )" />
-    <PackageVersion Include="Microsoft.OpenApi" Version="[1.6.3, )" />
-    <PackageVersion Include="Microsoft.OpenApi.Readers" Version="[1.6.3, )" />
+    <PackageVersion Include="Microsoft.OpenApi" Version="1.6.11" />
```

Comment:
@SergeyMenshykh is there any special reason for those packages being locked at `1.6.3` ? I can revert this if there's something I'm unaware.

Reply:
This syntax - [1.6.3, ) doesn't lock the package and on the other hand allows the package be automatically updated to any compatible (minor version or patch update) version without manual intervention. I've traced the change to this PR - https://github.com/microsoft/semantic-kernel/pull/446 and I don't see any particular reason except a try to get package automatically updated.